### PR TITLE
Update Aryn Cloud page

### PR DESCRIPTION
**Pull request by jonfritz**
Update Aryn Cloud page and URL

### CODE REVIEW

File: aryn_playground_cloud_console.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Using the Playground and Console
+description: "Using the Aryn Playground and Aryn Cloud Console"
+---
+
+## Aryn Playground
+
+The quickest way to experiment with the DocPrep and DocParse UIs is through the [Aryn Playground](https://play.aryn.cloud). You can access the Playground with just an ARyn Cloud API key.
```
Comment (karanataryn) [inline aryn_playground_cloud_console.mdx:8]: Typo: Aryn

Comment (karanataryn) [inline aryn_playground_cloud_console.mdx:8]: experiment with DocPrep and DocParse is through the Aryn Playground UIs?

Comment (jonfritz) [reply aryn_playground_cloud_console.mdx]: Ack

Comment (jonfritz) [reply aryn_playground_cloud_console.mdx]: Ack

File: aryn_playground_cloud_console.mdx [hunk 2]
```diff
@@ -5,7 +5,7 @@ description: "Using the Aryn Playground and Aryn Cloud Console"
 
 ## Aryn Playground
 
-The quickest way to experiment with the DocPrep and DocParse UIs is through the [Aryn Playground](https://play.aryn.cloud). You can access the Playground with just an ARyn Cloud API key.
+The quickest way to experiment with the DocPrep and DocParse is through the [Aryn Playground](https://play.aryn.cloud) UI. You can access the Playground with just an Aryn Cloud API key.
```
Comment (karanataryn) [inline aryn_playground_cloud_console.mdx:8]: typo: `DocPrep and DocParse` not `the DocPrep and DocParse`

**Review by karanataryn — APPROVED** [review]

**Pull request merged**